bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | rev | cut -d',' -f1 | rev )
```

Comment:
Maybe I am not so great at `cut`, but it isn't clear to me that there is an easy way to take _just the last_ item from a comma separated list using `cut`. However, you can fake this by reversing the list, taking _the first_ item, and reversing the list again.